<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-3</y>
     <width>750</width>
     <height>568</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>65</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2 {
background-color: #283f59;
}
</string>
    </property>
    <widget class="QPushButton" name="user">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>65</width>
       <height>59</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/user-svgrepo-com.svg</normaloff>../images/user-svgrepo-com.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>66</y>
       <width>65</width>
       <height>59</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border-top:2px solid white; </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/search-svgrepo-com.svg</normaloff>../images/search-svgrepo-com.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="history">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>126</y>
       <width>65</width>
       <height>59</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: transparent;
border-top: 2px solid white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/history-svgrepo-com.svg</normaloff>../images/history-svgrepo-com.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="plus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>65</width>
       <height>59</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: transparent;
border-top: 2px solid white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/plus.png</normaloff>../images/plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="all_items">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>65</width>
       <height>59</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: transparent;
border-top: 2px solid white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/shopping-bag.png</normaloff>../images/shopping-bag.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>34</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="change" native="true">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>0</y>
      <width>685</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#operations{
background-color: rgba(0,0,0,0.2);
}

</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
